src/Entity/ReferentTag.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity(repositoryClass="AppBundle\Repository\ReferentTagRepository")
+ * @ORM\Table(
+ *   name="referent_tags",
+ *   uniqueConstraints={
+ *     @ORM\UniqueConstraint(name="referent_tag_name_unique", columns="name"),
```

Comment:
It's simpler to add the unique constrain on the property with `unique=true`.
cf. https://www.doctrine-project.org/projects/doctrine-orm/en/latest/reference/annotations-reference.html#annref_column